apps/passport-client/components/screens/AddScreen/ProveAndAddScreen.tsx

```diff
@@ -42,7 +45,13 @@ export function ProveAndAddScreen({
       />
     );
   } else {
-    content = <AddedPCD />;
+    content = (
+      <AddedPCD
+        onCloseClick={() => {
+          safeRedirect(request.returnUrl, serializedPCD);
```

Comment:
fix looks good to me.

are you sure this was introduced by #224 tho? 

before that change, we were also redirecting immediately after `add-pcd`

<img width="632" alt="image" src="https://user-images.githubusercontent.com/169280/233787530-adb80475-1b61-4e5b-8a28-23e195f47de3.png">